.github/workflows/deploy.yml

```
@@ -143,14 +166,27 @@ jobs:
           done
           echo "Deployment started in ${SECONDS} seconds" >> $GITHUB_STEP_SUMMARY
           echo "::endgroup::"
-      - name: ✅ Update deployment status
-        if: success()
-        uses: chrnorm/deployment-status@v2
+      - name: 🔄 Update deployment status
```

actionlint (1.7.11)

[error] step must run script with "run" section or run action with "uses" section

(syntax-check)